File: appium-c-sharp/AppiumBasicSetup/BasicTest.cs
using NUnit.Framework;
using System;
using OpenQA.Selenium.Appium.Android;
using OpenQA.Selenium.Remote;
using OpenQA.Selenium.Appium;

namespace AppiumBasicSetup
{
	[TestFixture]
	public class BasicTest
	{
		AndroidDriver<AppiumWebElement> driver;

		[SetUp()]
		public void SetUp()
		{
			Console.Write("Setting up the test");
			DesiredCapabilities capabilities = new DesiredCapabilities();
			capabilities.SetCapability("testobject_api_key", "my_api_key");
			capabilities.SetCapability("testobject_device", "Samsung_Galaxy_S6_real");
			Uri server = new Uri("url_given_in_testobject_project");

			driver = new AndroidDriver<AppiumWebElement>(server, capabilities);
		}

		[Test()]
		public void FirstTest()
		{
			driver.GetScreenshot();
		}

		[TearDown()]
		public void TearDown()
		{
			driver.Quit();
			Console.Write("Tearing down the test");
		}
	}
}

File: appium-java/src/test/java/BasicTest.java
import org.junit.After;
import org.junit.Before;
import org.junit.Test;
import io.appium.java_client.android.AndroidDriver;
import org.openqa.selenium.remote.DesiredCapabilities;
import java.net.URL;


public class BasicTest {

    private AndroidDriver driver;

    @Before
    public void setUp() throws Exception {
        DesiredCapabilities capabilities = new DesiredCapabilities();
        capabilities.setCapability("testobject_api_key", "my_api_key");
        capabilities.setCapability("testobject_device", "Samsung_Galaxy_S6_real");
        URL server = new URL("url_given_in_testobject_project");

        driver = new AndroidDriver(server, capabilities);
    }

    @Test
    public void firstTest() throws Exception {
        System.out.println("Driver set up properly");
    }

    @After
    public void tearDown() throws Exception {
        driver.quit();
    }
}
